### PR TITLE
Stories and SSL Configuration docs update

### PR DESCRIPTION
**Pull request by ehconitin**

### CODE REVIEW

File: packages/twenty-front/src/modules/settings/components/SettingsOptions/__stories__/SettingsOptionCardContentToggle.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
+import { SettingsOptionCardContentToggle } from '@/settings/components/SettingsOptions/SettingsOptionCardContentToggle';
```
Comment (FelixMalfait) [inline packages/twenty-front/src/modules/settings/components/SettingsOptions/__stories__/SettingsOptionCardContentToggle.stories.tsx:1]: Did you bring in files from a different PR? thanks

Comment (ehconitin) [reply packages/twenty-front/src/modules/settings/components/SettingsOptions/__stories__/SettingsOptionCardContentToggle.stories.tsx:1]: https://github.com/twentyhq/twenty/pull/8456
Yes! My last PR was merged, but no stories! so I added them here.

File: packages/twenty-front/src/modules/settings/components/SettingsOptions/__stories__/SettingsOptionCardContentCounter.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    maxWidth: 800,
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof SettingsOptionCardContentCounter>;
+
+export const Default: Story = {
+  render: (args) => (
+    <CounterWithState
```
[the remark this answers is not in the document]
Comment (charlesBochet) [reply packages/twenty-front/src/modules/settings/components/SettingsOptions/__stories__/SettingsOptionCardContentCounter.stories.tsx:55]: and use decorator if you need additional behaviors such as states

**Review by charlesBochet — COMMENTED** [review]
I have left a comment which I think is important! Could you take a look?

File: packages/twenty-front/src/modules/settings/components/SettingsOptions/__stories__/SettingsOptionCardContentCounter.stories.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ehconitin) [reply packages/twenty-front/src/modules/settings/components/SettingsOptions/__stories__/SettingsOptionCardContentCounter.stories.tsx]: Done :)

**Review by charlesBochet — APPROVED** [review]
Thank you!

**Pull request merged**